### PR TITLE
Add FLOAT64-IEEE support to inputs.modbus (#8361) (by @Nemecsek)

### PR DESCRIPTION
**Pull request by soerman**
This adds FLOAT64-IEEE support (issue #8361). 
**Please note that @Nemecsek is the original author** &rarr;  See discussion in #8361.

### Required for all PRs:

- [x] Signed [CLA](https://influxdata.com/community/cla/).
- [x] Associated README.md updated.
- [x] Has appropriate unit tests.

### CODE REVIEW

**Comment by garciaolais** [comment]
@soerman please consider adding some test cases for this new type

**Comment by srebhan** [comment]
We also need the description in the `README.md` file. Futhermore, you need a CLA otherwise we cannot merge this PR...

File: plugins/inputs/modbus/modbus.go [hunk 1]
```diff
@@ -662,6 +667,10 @@ func scaleFloat32(s float64, v float32) float32 {
 	return float32(float64(v) * s)
 }
 
+func scaleFloat64(s float64, v float64) float64 {
+	return float64(float64(v) * s)
```
Comment (ssoroka) [inline plugins/inputs/modbus/modbus.go:671]: Everything's a float64 here, so just 
```suggestion
	return v * s
```

**Review by ssoroka — APPROVED** [review]

**Review by ssoroka — CHANGES_REQUESTED** [review]
Actually, we should have some test cases to avoid any regressions

**Comment by youcann** [comment]
Seems good now

**Review by srebhan — APPROVED** [review]
LGTM.

**Review by ssoroka — APPROVED** [review]

**Pull request merged**